### DISCUSSION TITLE
Export images showing location of solids and thermal sources

### DISCUSSION THREAD

**Discussion by aknsj** [ansys/pyaedt#5106] [Q&A]
Hi all, I used pyaedt to export model pictures and temperature pictures during post-processing, which can be used in my simulation reports, which is very helpful.
But I recently ran into a problem, and that problem is that I want to show in the report where in the model the solid or heat source is located.
When operating the software manually, I usually do as follow to achieve a localization map. In the model tree or heat source tree, I left click with the mouse on the solid or heat source I want to select. Then in the AEDT software interface, the selected solid turns dark purple and the selected heat source turns blue mesh. I then manually take a screenshot of the displayed image as shown below.
![image](https://github.com/user-attachments/assets/57da623c-3c78-4670-a2de-6e6883f35509)
![image](https://github.com/user-attachments/assets/e1b664b1-ea36-494d-a3fb-4db571beda1c)
![image](https://github.com/user-attachments/assets/b35eb376-0072-464e-865f-5083bb176ecb)
![image](https://github.com/user-attachments/assets/a343e333-832f-477e-ac96-c99bba23de78)
Since I need to inform the client of the location of solids and heat sources in my simulation reports, I'd be happy to know if I can export a picture of the location using the pyaedt code.Or there are other ways to show the localization of solids and heat sources, such as labeling.
The version of Pyaedt I am using is 0.9.10. looking forward to your reply, thanks!

**Comment by lorenzovecchietti** [comment]
To have full flexibility, you can use pyVista. An example to add annotation to the plot:
![immagine](https://github.com/user-attachments/assets/fb269b20-362f-4909-8714-dd11cfe3e92b)

(sorry if it is a screenshot, the example will be public soon)
